public_html/wp-content/plugins/wordcamp-payments-network/includes/sponsor-invoices-dashboard.php

```diff
@@ -461,3 +464,107 @@ function delete_index_row( $invoice_id ) {
 		)
 	);
 }
+
+/**
+ * Schedule cron to send reminder mails to organizers for unpaid invoices.
+ */
+function schedule_sent_invoice_reminder() {
+	if ( wp_next_scheduled( 'send_invoice_pending_reminder' ) ) {
+		return;
+	}
+
+	wp_schedule_event( time(), 'daily', 'send_invoice_pending_reminder' );
+}
+
+/**
+ * Send reminder to organizer about the unpaid invoice.
+ */
+function send_invoice_pending_reminder() {
+	global $wpdb;
+
+	$table_name    = get_index_table_name();
+	$sent_invoices = $wpdb->get_results( "
+		SELECT blog_id, invoice_id
+		FROM $table_name
+		WHERE status = 'wcbsi_approved'
+		LIMIT 1000
+	" );
+
+	foreach ( $sent_invoices as $invoice_data ) {
+		$blog_id    = $invoice_data->blog_id;
+		$invoice_id = $invoice_data->invoice_id;
+
+		switch_to_blog( $blog_id );
+
+		$invoice         = get_post( $invoice_id );
+		$invoice_sent_at = get_post_meta( $invoice_id, 'Sent at', true );
+
+		if ( empty( $invoice_sent_at ) ) {
+			// Backfill for older invoices.
+			update_post_meta( $invoice_id, 'Sent at', time() );
+			update_post_meta( $invoice_id, 'Backfilled Sent at', true );
+			restore_current_blog();
+			continue;
+		}
```

Comment:
Just a thought: you could get more accurate sent dates for the backfill using our QBO API client, like is used in the Sponsor Invoices report:

https://meta.trac.wordpress.org/browser/sites/trunk/wordcamp.org/public_html/wp-content/plugins/wordcamp-reports/classes/report/class-sponsor-invoices.php#L274

Reply:
Since its not a very strict requirement, and we are anyways defaulting the invoice after 60 days since WordCamp, I'd rather not do this extra work which we are not gonna use in future. Do you think there perhaps could be a good reason to prefill accurately that I am not thinking of?

Reply:
> Do you think there perhaps could be a good reason to prefill accurately that I am not thinking of?

Nope. If you don't think the accuracy is necessary here, then I'm not concerned about it either :)